packages/now-static-build/src/index.ts

```diff
@@ -1,6 +1,5 @@
 import ms from 'ms';
 import path from 'path';
-import spawn from 'cross-spawn';
```

Comment:
@AndyBitz  Since you removed `cross-spawn` import, can you also uninstall the dependency? 🤔